libraries/xcode_command_line_tools.rb

```diff
@@ -154,7 +154,7 @@ def attach
     # @return [void]
     #
     def install
-      execute %|installer -package "$(find '#{mount_path}' -name *.mpkg)" -target "/"|
+      execute %|installer -allowUntrusted -package "$(find '#{mount_path}' -name *.mpkg)" -target "/"|
```

Comment:
The `-allowUntrusted` flag should only be included with the affected platform version so `10.7` in this case. We want the default behavior to only allow trusted packages to be installed. You'll also want to add a big comment explaining why the `-allowUntrusted` flag is being used on these versions...maybe something like:

```ruby
# The "-allowUntrusted" flag has been added to the installer
# command to accommodate for now-expired certificates used
# to sign the downloaded command line tools.
```